util.c

```diff
@@ -5457,3 +5457,40 @@ void util_set_hostname(char *hostname)
   log_info("set hostname: %s\n", hostname);
 }
 
+
+/*
+ * Run debug shell.
+ *
+ * This takes care of screen settings and restoring standard file
+ * descriptors.
+ */
+void util_run_debugshell()
+{
+  kbd_end(1);
```

Comment:
Really kbd_end(1) ? In both code blocks where this is copied from, it's kbd_end(0).